src/main/kotlin/racingcar/Track.kt

```diff
@@ -3,36 +3,60 @@ package racingcar
 import racingcar.io.InputView
 import racingcar.io.ResultView
 
-class Track(private val random: RandomStrategy) {
+class Track(private val random: ForwardStrategy) {
 
     fun play() {
-        ResultView.printMessage(ResultView.Message.CAR_COUNT)
-        val carCount = InputView.requestPositiveNumber()
-        val cars: List<Car> = (1..carCount).map { Car() }
+        ResultView.printMessage(ResultView.Message.CAR_NAMES_WITH_COMMA)
+        val names: List<String> = InputView.requestString(DELIMITERS)
+        val cars: List<Car> = names.map { Car(name = it) }
 
         ResultView.printMessage(ResultView.Message.PLAY_COUNT)
-        val playCount = InputView.requestPositiveNumber()
+        val playCount: Int = InputView.requestPositiveNumber()
 
         ResultView.printMessage(ResultView.Message.RESULT)
-        this.start(playCount = playCount, cars = cars)
+        val finishedRacingCars: List<Car> = this.start(playCount, cars)
+        val winners: String = this.getWinnerNames(finishedRacingCars)
+
+        ResultView.printMessage(winners, ResultView.Message.WINNER)
     }
 
-    private fun start(playCount: Int, cars: List<Car>) =
-        repeat(playCount) {
-            this.move(cars)
-            this.printResult(cars)
-        }
+    private fun getWinnerNames(cars: List<Car>): String =
+        this.getWinner(cars).joinToString(",") { it.name }
+
+    fun getWinner(cars: List<Car>): List<Car> {
+        val carsByPosition = cars.groupBy { it.position }
+        val maxPosition: Int = carsByPosition.maxOf { it.key }
+
+        return carsByPosition[maxPosition] ?: emptyList()
+    }
```

Comment:
List<T>?.orEmpty() 라는 확장함수를 활용해보는 것은 어떨까요?
```suggestion
        return carsByPosition[maxPosition].orEmpty()
    }
```

Reply:
이번에 처음 알게 되었습니다. 감사합니다. 😄 👍